### PR TITLE
add demographic dimensions (#34)

### PR DESCRIPTION
**Pull request by GeorgLink**
adding:
* Romantic Orientation
* Religion

### CODE REVIEW

**Comment by GeorgLink** [comment]
fixes #34

File: demographic-data/README.md [hunk 1]
```diff
@@ -25,17 +25,19 @@ NOTE: that *how* we might ask these questions, and what potential answers might
 
 1. Gender Identity
 2. Sexual orientation
-3. Age
-4. Location/Region/Country
+3. Romatic orientation
```
Comment (Erioldoesdesign) [inline demographic-data/README.md]: Spell check on 'Romantic'

Comment (GeorgLink) [reply demographic-data/README.md]: Thanks. I fixed it.

**Comment by GeorgLink** [comment]
I changed them to have all "1." in front of them because then a commit would only need to edit one line if we added a characteristic in the middle of the list. The problem with numbering them subsequentially is that inserting an entry requires changing all the following lines and it muddies the commit history.

Having all entries with "1." is proper Markdown formatting.

**Comment by germonprez** [comment]
Cool! Thanks for the detail @GeorgLink

**Pull request merged**